README.md

```diff
@@ -1,7 +1,60 @@
 # joy
 
-## Development
+Joy is a GitOps CLI tool with CRD-style YAML file formats for simplifying the deployment of project releases to Kubernetes and their promotion across environments. It proves a perfect complement to other GitOps tools, such as [ArgoCD](https://argoproj.github.io/cd/) and [Crossplane](https://www.crossplane.io/).
 
-### Run commands without compiling
+- Defines CRD-style YAML file formats:
+  - `Environment`: A Kubernetes cluster and/or namespace to deploy to.
+  - `Project`: An application (eg: a microservice, front-end...) that can be deployed to different environments.
+  - `Release`: The deployment configuration for a specific version of a project to a given environment.
+- GitOps-oriented: Your joy "catalog" git repo defines the Source of Truth™ for your environments, projects and releases. Changes to releases can be detected by a CD tool such as ArgoCD, transformed and deployed to the target environment.
```

Comment:
```suggestion
- GitOps-oriented: Your joy "catalog" git repo defines the Source of Truth™ for your environments, projects and releases. Combined with the ApplicationSet generator plugin, changes to releases are detected by ArgoCD, transformed and deployed to the target environment.
```

Considering we don't have other methods of deploying the releases at the moment, I find this wording more clear.